core/common/src/annotation/TextBlock.ts

```diff
@@ -497,7 +559,9 @@ export class TextBlock extends TextBlockComponent {
     return TextBlock.create({ styleName: "" });
   }
 
-  /** Returns true if every paragraph in this text block is empty. */
+  /** Returns true if every paragraph in this text block is empty.
+   * idk, should I get rid of this? I can't see a use for it. using isWhitespace is probably better in most cases.
+  */
```

Comment:
I need to remove this comment, but the question is still valid. Is isEmpty useful? I mean it corresponds to createEmpty, but it shouldn't be used when determining whether or not geometry can be created. That would rely on `isWhitespace`

Reply:
Okay, I can leave it then and remove the comment